app/assets/v2/js/pages/bounty_details.js

```diff
@@ -859,6 +859,57 @@ var show_extend_deadline_modal = function() {
   });
 };
 
+const appendGithubSyncButton = function(result) {
+  if (isBountyOwner(result) || currentProfile.isStaff) {
+    const title = gettext('Sync Issue');
+
+    $('#github_actions').append('<button sync-github-issue id="btn-white" type="button" class="btn btn-small">' + title + '</button>');
+
+    $('#github_actions [sync-github-issue]').on('click', function(e) {
+      const bountyId = result.pk;
+
+      e.preventDefault();
+
+      $.get(
+        '/sync/get_issue_details?url=' + encodeURIComponent(result['github_url']) + '&token=' + currentProfile.githubToken
+      ).then(function(result) {
+        const payload = JSON.stringify({
+          issue_description: result.description
+        });
+
+        return $.post('/bounty/change/' + bountyId, payload).then(
+          function(result) {
+            result = sanitizeAPIResults(result);
+            _alert({ message: result.msg }, 'success');
+
+            if (result.url) {
+              setTimeout(function() {
+                document.location.href = result.url;
+              }, 1000);
+            }
+          }
+        ).fail(
+          function(result) {
+            var alertMsg = result && result.responseJSON ? result.responseJSON.error : null;
+
+            if (alertMsg === null) {
+              alertMsg = gettext('Failed to sync issue. Please reload the page and try again.');
+            }
+            _alert({ message: alertMsg }, 'error');
+          }
+        );
+      }).fail(function(result) {
+        var alertMsg = result && result.responseJSON ? result.responseJSON.error : null;
+
+        if (alertMsg === null) {
```

Comment:
let alertMsg = result && result.responseJSON ? result.responseJSON.error : gettext('Failed to sync issue. Please reload the page and try again.');